### DISCUSSION TITLE
how to use this model for text summarization

### DISCUSSION THREAD

**Discussion by kenzaRe** [aub-mind/arabert#165] [Q&A] [answered]
Hello!
I want to use this model for Arabic text summarization. How should I fine-tune it?
Thank you.

**Comment by WissamAntoun** [comment, marked as answer]
AraBERT can't be used for generative summarization, you can probably do "extractive " where you rank each sentence if it is worthy to be included in the summary,

You should check AraGPT2, and fine tune it for generative summarization, using any existing arabic summerization dataset which you can check here https://arbml.github.io/masader/

**Reply by kenzaRe** [reply]
Thank you, but how would I know if a model can be used for generative summarization?

**Reply by WissamAntoun** [reply]
BERT and Electra can't do text generation, while GPT2 models can.
